consultation_analyser/settings/base.py

```diff
@@ -190,3 +190,6 @@
 GOVUK_NOTIFY_API_KEY = env("GOVUK_NOTIFY_API_KEY")
 GOVUK_NOTIFY_PLAIN_EMAIL_TEMPLATE_ID = env("GOVUK_NOTIFY_PLAIN_EMAIL_TEMPLATE_ID")
 USE_SAGEMAKER_LLM = env.bool("USE_SAGEMAKER_LLM")
+
+# Authentication
+LOGIN_URL = "/sign-in"
```

Comment:
Please can we be consistent and end all URLs with `/`?

Reply:
YES we can — thank you 💯